src/PhenX.EntityFrameworkCore.BulkInsert.SqlServer/SqlServerDialectBuilder.cs

```diff
@@ -88,7 +119,8 @@ public override string BuildMoveDataSql<T>(
             q.AppendLine($"SET IDENTITY_INSERT {target.QuotedTableName} OFF;");
         }
 
-        return q.ToString();
+        var x = q.ToString();
```

Comment:
not necessary

Reply:
it was easier to debug like that, I have forgotten to give a proper name to the variable.

Reply:
I have renamed it to "result"